src/components/activity-completion/summary-table.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import IconComplete from "../../assets/svg-icons/icon-check-circle.svg";
+import IconIncomplete from "../../assets/svg-icons/icon-unfinished-check-circle.svg";
+
+import "./summary-table.scss";
+
+interface IProps {
+  questionsStatus: Array<{number: number, page: number, prompt: string, answered: boolean}>;
+}
+
+export const SummaryTable: React.FC<IProps> = (props) => {
+  const { questionsStatus } = props;
+
+  return (
+    <table className="summary-table" data-cy="summary-table">
+      <thead>
+        <tr>
+          <th>Question</th>
+          <th>Complete</th>
+        </tr>
+      </thead>
+      <tbody>
+      {questionsStatus.map((question: {number: number, page: number, prompt: string, answered: boolean}, index) => {
```

Comment:
You could extract the interface for a single `interface IQuestionStatus {number: number, page: number, prompt: string, answered: boolean}` and then make `interface IProps { questionsStatus: Array< IQuestionStatus> }` — You might then be able to reuse `IQuestionStatus` in `completion-page-content.tsx`.  Just a thought.